remote_write.go

```diff
@@ -28,16 +28,39 @@ import (
 // Register the extension on module initialization, available to
 // import from JS as "k6/x/remotewrite".
 func init() {
-	modules.Register("k6/x/remotewrite", new(RemoteWrite))
+	modules.Register("k6/x/remotewrite", new(root))
 }
 
 // RemoteWrite is the k6 extension for interacting Prometheus Remote Write endpoints.
-type RemoteWrite struct{}
+type RemoteWrite struct {
+	vu modules.VU
+}
+
+type root struct{}
```

Comment:
I don't know if there are any conventions around the naming of this `root` type which I'm not aware of, but when I first looked at it I didn't know what `root` is, so I'd consider renaming it to something more explanatory like `remoteWriteModule`